server/fixtures/article.js

```diff
@@ -5,11 +5,15 @@ const slugTitle = createUniqueSlug('hello world');
 export const newArticle = {
   title: 'Hello world!',
   body: 'Just a dummy data',
-  userId: 'a5b37ee6-1553-4681-a343-d8296344e129',
+  userId: 'f1e2e1c8-6a90-4bcd-b349-49f989d311d6',
   isBlacklisted: false,
   slug: slugTitle,
   categoryId: '325163a1-6e8c-4cd8-b5d3-7019c49983f5',
   isPublished: true
 };
 
-export default { newArticle };
+export const article = {
```

Comment:
i think you can use newArticle instead of creating another article

Reply:
I agree with @mosinmiloluwa01, do you mind implementing the feedback or communicating your reasons before Resolving conversation